src/main/java/com/example/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,32 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
for now, it may be fine but in the future, it is not recommended to use `@Data` with `@Entity`
more details on that: https://medium.com/@dev.fethi/data-anatation-and-hibernate-n-1-problems-466df982288e